scripts/reports/repo-health.sh

```diff
@@ -31,6 +31,14 @@ yum install gcc python3-devel git-lfs pango-devel -y
 pip3 install seaborn matplotlib pandas xlrd requests python-dateutil pyGithub jupyterlab django-weasyprint weasyprint==52.5
 git lfs install
 
+#Install alex
+yum install epel-release -y; curl https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+source ~/.bashrc
+nvm --version
+nvm install 14.20.0
+npm install alex --global -y
+alex --version
```

Comment:
Please add some echo statements so it will be easy to search in the logs

Reply:
Okay

Reply:
Let me know what more echo statements are required.

Reply:
You can fetch and print "Node Version: $NODE_VERSION  Alex Version: $ALEX_VERSION"